compile.js

```diff
@@ -0,0 +1,59 @@
+qx.Class.define("qx.compiler.LibraryApi", {
```

Comment:
Is'nt this test code? If yes it should be somwhere in the test directory.

Reply:
It is, but you don't run any `qx` in the repo root of qooxdoo-compiler repo unless you are doing some testing or development - so I don't think it hurts. Tests will only be run when `qx test` is executed.

Reply:
i run qx lint in the repo root!

Reply:
Ideally, this would run "real" tests, i.e. the tests in the test dir. These are just stubs to test the functioning of the "internal" test runner itself. Should be replaced with something real eventually.

Reply:
ok, agreet.